run

```diff
@@ -4,6 +4,15 @@ export NATS_URL=nats://localhost:4222
 export ELASTICSEARCH_URI=http://localhost:9200
 export ELASTICSEARCH_INDEX_NAME=jwa_item_store
 
+set -euo pipefail
+
+declare -a tags=(
+    $(git rev-parse --verify HEAD --short)
+    $(cat ./package.json | docker run --rm -i imega/jq -r '.version')
```

Comment:
Do we really need to launch a container to do that ? i know perf isn't critical here but it seems a little overkill to me ^^

Reply:
is there another tool that's standard in our bash/zsh that I can use? I like jq but don't feel like asking all devs to install it to run those scripts. What do you think?